streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordDeserializer.java

```diff
@@ -70,7 +70,10 @@
         } catch (final Exception deserializationException) {
             final DeserializationExceptionHandler.DeserializationHandlerResponse response;
             try {
-                response = deserializationExceptionHandler.handle(processorContext, rawRecord, deserializationException);
+                response = deserializationExceptionHandler.handle(
+                    (InternalProcessorContext<?, ?>) processorContext,
```

Comment:
Oy. Do you mind filing a `needs-kip` Jira to migrate DeserializationExceptionHandler to the new API?

Reply:
Thanks!